content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md

```diff
@@ -46,6 +48,10 @@ The fields are described as follows:
 
 * `name`: Name of the Eip object.
 
+* `annotations`:
+
+  * `eip.openelb.kubesphere.io/is-default-eip`: Whether the current Eip object is the default Eip object. The value can be `"true"` or `"false"`. If a default Eip object exists, you do not need to specify an Eip object when creating a Service and the system automatically assigns an IP address from the default Eip object to the Service. If you do not specify an Eip object when creating a Service while no default IP address exists, the system automatically assigns an IP address from any available Eip object, if any, to the Service. For each Kubernetes cluster, you can set only one Eip object as the default Eip object.
```

Comment:
When `eip.openelb.kubesphere.io/is-default-eip` is `true`, there is no need to add additional annotations to `Service`,for example
``` 
annotations:
  lb.kubesphere. io/v1alpha1: openelb
  protocol. openelb. kubesphere. io/v1alpha1: layer2
  eip. openelb. kubesphere. io/v1alpha2: layer2
```
these annotations are not required